ddht/v5_1/alexandria/network.py

```diff
@@ -64,6 +199,55 @@ async def _pong_when_pinged(self) -> None:
                     request_id=request.request_id,
                 )
 
+    async def _serve_find_nodes(self) -> None:
+        async with self.client.subscribe(FindNodesMessage) as subscription:
+            async for request in subscription:
+                response_enrs: List[ENRAPI] = []
+                distances = set(request.message.payload.distances)
+                if len(distances) != len(request.message.payload.distances):
+                    self.logger.debug(
+                        "Ignoring invalid FindNodesMessage from %s@%s: duplicate distances",
+                        humanize_node_id(request.sender_node_id),
+                        request.sender_endpoint,
+                    )
```

Comment:
Makes sense but I wish we had a way of telling the other node what they did wrong. This seems like a weak violation of Postel's law, does the spec say we should ignore these messages?